### PR TITLE
Added mentors Issue: #96

### PR DESCRIPTION
**Pull request by Sid911**
added 1 more mentors to the mentors.yml

### CODE REVIEW

**Comment by Sid911** [comment]
@fauh45 done sir 😊

File: _includes/header.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
               <li><a href="#welcome">Welcome</a></li>
               <li><a href="#mentors">Mentors</a></li>
               <li><a href="#students">Students</a></li>
-              <li><a href="#projects">Projects</a></li>
+              <li><a href="/projects.html">Projects</a></li>
```
Comment (jogendra) [inline _includes/header.html]: this is not related to issue

File: index.html [hunk 2]
```diff
@@ -109,10 +109,10 @@ <h3>What is FOSSASIA?</h3>
 	<!--end of container-->
 	</section>
 	<section>
-    <div class="container">
+    <div class="container" id="students">
         <div class="row mb64 mb-xs-24" style="margin-top: 50px">
             <div class="col-sm-12 col-md-10 col-md-offset-1 text-center">
-                <h3> Students </h3>
+                <h2> Students </h2>
```
Comment (jogendra) [inline index.html]: same here

File: _includes/header.html (hunk 1 above)
Comment (Sid911) [reply _includes/header.html]: @jogendra Its actually of the master brach of my fork so, it was included. How Can I change that?

**Comment by fragm3** [comment]
@Sid911 remove all the changes from index.html file, not needed

**Comment by STEdirisuriya** [comment]
Squash commits

**Comment by Sid911** [comment]
@fragm3 how do I delete commits?

**Comment by jogendra** [comment]
@Sid911 Look like you have created new branch from your another PR branch instead of master. Make a new PR keeping that in mind. thanks
